Directory.Packages.props

```diff
@@ -120,5 +120,8 @@
     <PackageVersion Include="Microsoft.DotNet.Build.Tasks.Installers" Version="8.0.0-beta.23564.4" />
     <!-- unit test dependencies -->
     <PackageVersion Include="Microsoft.Extensions.Diagnostics.Testing" Version="8.0.0" />
+    <!-- Pinned version for Component Governance - Remove when root dependencies are updated -->
```

Comment:
Would it be useful to open an issue to remind us of doing that?

Reply:
GitHub feature request: scheduled issues, we need a bot. "Hey GitHub, remind me to check the dependencies in one month"

Reply:
Sounds like a great little idea for a sample Aspire application that processes GitHub webhooks :)